packages/sirius-web/backend/sirius-web-services/src/main/java/org/eclipse/sirius/web/services/representations/ViewRepresentationDescriptionSearchService.java

```diff
@@ -144,6 +149,28 @@ public Optional<EdgeDescription> findViewEdgeDescriptionById(String edgeDescript
         return Optional.empty();
     }
 
+    @Override
+    public Optional<FormElementDescription> findViewFormElementDescriptionById(String formDescriptionId) {
+        Optional<String> sourceId = this.getSourceId(formDescriptionId);
+        Optional<String> sourceElementId = this.getSourceElementId(formDescriptionId);
+        if (sourceId.isPresent() && sourceElementId.isPresent()) {
+            List<View> views = this.getViewsFromSourceId(sourceId.get());
+            if (!views.isEmpty()) {
+                // @formatter:off
+                var searchedViewNodes = views.stream()
```

Comment:
I'm pretty sure you are not returning diagram nodes here. Please be careful while copying and pasting code around